### ISSUE TITLE
Docker timeout when starting

### ISSUE THREAD

**Issue by furio** [docker/for-win#581]
### Expected behavior
Docker have to start

### Actual behavior
Docker doesn't start error is provided:
```
Docker command did not complete within the allowed timeout
   in Docker.Backend.MobyCommand.RunDocker(String arguments, IDictionary`2 envVariables, String stdin, Int32 timeoutMs, String cidFile)
   in Docker.Backend.MobyCommand.Run(String commandLine, IDictionary`2 envVariables, Boolean load, String stdin, Int32 timeoutMs)
   in Docker.Backend.SambaShare.MountInMoby(String mount, String drive, Credential cred, CredentialInfo credInfo, String options, String ip)
   in Docker.Backend.SambaShare.MountInMoby(String drive, Credential credential, Settings settings)
   in Docker.Backend.SambaShare.Mount(String drive, Credential credential, Settings settings)
   in Docker.Backend.BackendServer.<Run>b__5_11(Object[] args)
   in Docker.Core.Pipe.NamedPipeServer.RunAction(String action, Object[] parameters)
```

### Information

  - Diagnostic ID from "Diagnose & Feedback" in the menu.
  - a reproducible case if this is a bug, Dockerfiles FTW
  - page URL if this is a docs issue or the name of a man page
  - host distribution and version (Windows version, build number, etc)

1C94AFFE-10D4-4B6B-A4D3-F16E0AFCA683/2017-03-14_10-54-18
Docker: Version 17.03.0-ce-win1 (10300) Channel: beta 05a8abd
Windows 10 Pro - 1607 - 14393.693
(A docker reset to factory seems to fix the issue)

### Steps to reproduce the behavior

  1. Start windows
  2. Start docker

**Comment by friism** [comment]
I'm sorry Docker broke for you, but great that you got it fixed with a reset.

**Comment by ilanc** [comment]
I had the same issue and fixed it by changing the DNS Server from Automatic to Fixed: 8.8.8.8
I noticed this in the error log:
```
[20:03:41.671][VpnKit         ][Warning] com.docker.slirp.exe: de2b Query:0 na:c:r:rn 0 <qs:1.65.168.192.in-addr.arpa. <PTR|IN>> <an:> <au:> <ad:> lookup failed: no servers configured
[20:03:44.172][VpnKit         ][Warning] com.docker.slirp.exe: de2b Query:0 na:c:r:rn 0 <qs:1.65.168.192.in-addr.arpa. <PTR|IN>> <an:> <au:> <ad:> lookup failed: no servers configured
```

**Comment by mbunge** [comment]
Thanks @ilanc! This works for me too. No Reset necessary!
